### DISCUSSION TITLE
More ship options

### DISCUSSION THREAD

**Discussion by BarryBend** [bcholmes/StarTrek2d20#217] [Ideas]
The program is missing factions for the ship builders. Also more integration with Continuing mission would also help.

**Comment by Felderburg** [comment]
You should be able to manually write in traits for a starship, which I think is the only place a "faction" would apply?

What sort of integration with CM would you want to see?

**Comment by BarryBend** [comment]
I was thinking ship class templates as I looked through continuing mission
and it's missing all the ones I'd like to see in a campaign. And while
there are ones I can find I'd like easy to work on templates.
